statsmodels/base/_parameter_inference.py

```diff
@@ -256,3 +256,45 @@ def score_test(self, exog_extra=None, params_constrained=None,
         return stat, pval
     else:
         raise NotImplementedError('only hypothesis "joint" is available')
+
+
+def im_ratio(results):
+    res = getattr(results, "_results", results)  # shortcut
+    hess = res.model.hessian(res.params)
+    if res.cov_type == "nonrobust":
+        score_obs = res.model.score_obs(res.params)
+        cov_score = score_obs.T @ score_obs
+        hessneg_inv = np.linalg.inv(-hess)
+        im_ratio = hessneg_inv @ cov_score
```

Comment:
use lstsqu to avoid direct matrix inversion

Reply:
If these are both always square and hess is full-rank, than `solve` would be the best solution I think.